feature/platform/tests/telemetry_inventory_test/README.md

```diff
@@ -19,30 +19,60 @@ card, power supply, disk, flash, NPU, transceiver, fabric card), validate:
+     /components/component/software-module/state/module-type
+     /components/component/state/mfg-date
+     /components/component/state/software-version
+
+```yaml
+paths:
+    /components/component/state/description:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "FAN", "LINECARD", "POWER_SUPPLY"]
+    /components/component/state/firmware-version:
+       platform_type: ["TRANSCEIVER"]
+    /components/component/state/hardware-version:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "LINECARD", "POWER_SUPPLY", "TRANSCEIVER"]
+    /components/component/state/id:
+       platform_type: ["CONTROLLER_CARD", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "SENSOR"]
+    /components/component/state/mfg-name:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "LINECARD", "POWER_SUPPLY", "TRANSCEIVER"]
+    /components/component/state/model-name:
+       platform_type: ["CHASSIS"]
+    /components/component/state/name:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "CPU", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "SENSOR", "STORAGE", "TRANSCEIVER"]
+    /components/component/state/oper-status:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "CPU", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "STORAGE", "TRANSCEIVER"]
+    /components/component/state/parent:
+       platform_type: ["CONTROLLER_CARD", "FABRIC", "LINECARD", "POWER_SUPPLY"]
+    /components/component/state/part-no:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "CPU", "FABRIC", "FAN", "LINECARD", "POWER_SUPPLY", "STORAGE", "TRANSCEIVER"]
+    /components/component/state/serial-no:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "CPU", "FABRIC", "FAN", "LINECARD", "POWER_SUPPLY", "STORAGE", "TRANSCEIVER"]
+    /components/component/state/type:
+       platform_type: ["CHASSIS", "CONTROLLER_CARD", "CPU", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "SENSOR", "STORAGE", "TRANSCEIVER"]
+    /components/component/state/temperature/alarm-status:
+       platform_type: ["SENSOR"]
+    /components/component/state/temperature/instant:
+       platform_type: ["SENSOR"]
+    /components/component/state/temperature/max:
+       platform_type: ["SENSOR"]
+    /components/component/state/temperature/max-time:
+       platform_type: ["SENSOR"]
+    /components/component/integrated-circuit/backplane-facing-capacity/state/available-pct:
+       platform_type: ["INTEGRATED_CIRCUIT"]
+    /components/component/integrated-circuit/backplane-facing-capacity/state/consumed-capacity:
+       platform_type: ["INTEGRATED_CIRCUIT"]
+    /components/component/integrated-circuit/backplane-facing-capacity/state/total:
+       platform_type: ["INTEGRATED_CIRCUIT"]
+    /components/component/integrated-circuit/backplane-facing-capacity/state/total-operational-capacity:
+       platform_type: ["INTEGRATED_CIRCUIT"]
+
+rpcs:
+  gnmi:
+    gNMI.Get:
```

Comment:
@dplore The only gnmi call that is made in the test is [GetAll()](https://github.com/openconfig/ygnmi/blob/a0fdb565dc4c80f8151bb0469285f1063d8a4fdc/ygnmi/ygnmi.go#L471), but I don't see an RPC for this. Do you know if this is just a use of the gNMI.Get()?

Reply:
Correct, it should be `gNMI.Get`